packages/create-qwik/create-interactive.ts

```diff
@@ -71,7 +71,13 @@ export async function runCreateInteractiveCli() {
     }
 
     if (existingOutDirAnswer === 'replace') {
-      removeExistingOutDirPromise = fs.promises.rm(outDir, { recursive: true });
```

Comment:
just tested it and it looks like we don't need the `removeExistingOutDirPromise` anymore. In the current implementation, after confirming the deletion, the directory is removed bevor the type of the app is selected.

this is now also the case with this implementation. imo i would wait with the deletion till the type (qwik city, library, docs page, etc) is selected but @adamdbradley pls advice on this 👼 

thanks again for the adjustments 🙏

Reply:
ps: this is also the reason why the CI isn't happy atm

Reply:
Oh, now that I look at it, there indeed was that. Should I assign new code back to this variable?

Reply:
imo we could remove it everywhere and just move the code execution below tbe type selection to give the user another 'escape hatch'. but i'd wait for @adamdbradley or @manucorporat s feedback tbh 😃